src/main/java/com/bringframework/factory/ConfigBeanProcessor.java

```diff
@@ -0,0 +1,66 @@
+package com.bringframework.factory;
+
+import static com.bringframework.util.BeanUtils.createInstance;
+
+import com.bringframework.registry.BeanDefinition;
+import com.bringframework.registry.BeanDefinitionRegistry;
+import com.bringframework.registry.ConfigBeanDefinition;
+import com.bringframework.resolver.DependencyResolver;
+import java.util.Arrays;
+import java.util.Map;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * Implementation of {@link BeanProcessor} that is responsible for processing {@link BeanDefinition}s
+ * of beans from classes marked as {@link com.bringframework.annotation.Configuration}
+ * provided  from {@link BeanDefinitionRegistry}.
+ *
+ * <p>ConfigBeanProcessor is not responsible for injecting other beans, but {@link AutowiredBeanPostProcessor}.
+ *
+ * @since 1.0
+ */
+@Slf4j
+@RequiredArgsConstructor
+public class ConfigBeanProcessor implements BeanProcessor {
+
+  private final BeanDefinitionRegistry beanDefinitionRegistry;
+  private final Map<String, Object> rawBeanMap;
+  private final DependencyResolver dependencyResolver;
+
+  /**
+   * {@inheritDoc}
+   */
+  public void process() {
+    Map<String, ConfigBeanDefinition> configBeanDefinitionMap =
+        beanDefinitionRegistry.getAllConfigBeanDefinitions();
+    var beansNumber = configBeanDefinitionMap.size();
+    log.debug("creating {} row config beans found in bean definition registry", beansNumber);
```

Comment:
Please, 
1. make sure log message has the same structure as in RawBeanProcessor
2. The sentence should start with capital letter
3. there is a typo in `row`, probably you mean `raw`